### PR TITLE
Polish CONTRIBUTING

### PR DESCRIPTION
**Pull request by SeboPG**

### CODE REVIEW

**Comment by SeboPG** [comment]
I had few problems when uploading this file because I dont use Git for a long time

**Comment by wopian** [comment]
The `Co mam sprawdzać?` section was recently updated and needs re-translating to match the current translation. 

You can find the changes made to the English CONTRIBUTING at https://github.com/chrislgarry/Apollo-11/commit/16aa2154b23dda648a2ab568734e11627af8448b

**Review by SeboPG — COMMENTED** [review]
Latest polish CONTRIBUTING

**Review by SeboPG — COMMENTED** [review]
The latest version

**Review by SeboPG — COMMENTED** [review]
The latest latest

File: CONTRIBUTING.pl.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+[English][EN],
+[Español][ES],
+[Français][FR],
+[Italiano][IT],
+[Kurdi][KU],
+[Lietuvių][LT],
+[Nederlands][NL],
+[Norsk][NO],
+[Português][PT_BR],
+**Polski**,
```
Comment (wopian) [inline CONTRIBUTING.pl.md:15]:
```suggestion
**Polski**,
[Português][PT_BR],
```

File: CONTRIBUTING.pl.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+[GR]:CONTRIBUTING.gr.md
+[IT]:CONTRIBUTING.it.md
+[JA]:CONTRIBUTING.ja.md
+[KO_KR]:CONTRIBUTING.ko_kr.md
+[KU]:CONTRIBUTING.ku.md
+[LT]:CONTRIBUTING.lt.md
+[NL]:CONTRIBUTING.nl.md
+[NO]:CONTRIBUTING.no.md
+[PT_BR]:CONTRIBUTING.pt_br.md
+[PL]:CONTRIBUTING.pl.md
```
Comment (wopian) [inline CONTRIBUTING.pl.md:39]:
```suggestion
[PL]:CONTRIBUTING.pl.md
[PT_BR]:CONTRIBUTING.pt_br.md
```

File: CONTRIBUTING.pl.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+Typowe problemy, na które należy zwrócić uwagę podczas sprawdzania, obejmują między innymi:
+
+#### Literówki
+
+W niektórych miejscach pierwotni programiści popełnili błędy typograficzne podczas pisania komentarzy. Niektóre z nich zostały błędnie poprawione podczas wstępnej digitalizacji, jednak digitalizacja wprowadziła również błędy typograficzne, których nie było w skanach.
+
+Na przykład, jeśli zdigitalizowane komentarze zawierały słowo „SPACECRAFT”, ale na skanach odczytano „SPAECRAFT”, wówczas digitalizacja **MUSI** zostać poprawiona na „SPAECRAFT” (brakujące „C”).
+
+Podobnie, jeśli słowo zawiera literówkę w digitalizacji, ale jest poprawnie napisane na skanach, wówczas literówka **MUSI** zostać poprawiona.
+
```
Comment (wopian) [inline CONTRIBUTING.pl.md:102]:
```suggestion

```

File: CONTRIBUTING.pl.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+W niektórych miejscach pierwotni programiści popełnili błędy typograficzne podczas pisania komentarzy. Niektóre z nich zostały błędnie poprawione podczas wstępnej digitalizacji, jednak digitalizacja wprowadziła również błędy typograficzne, których nie było w skanach.
+
+Na przykład, jeśli zdigitalizowane komentarze zawierały słowo „SPACECRAFT”, ale na skanach odczytano „SPAECRAFT”, wówczas digitalizacja **MUSI** zostać poprawiona na „SPAECRAFT” (brakujące „C”).
+
+Podobnie, jeśli słowo zawiera literówkę w digitalizacji, ale jest poprawnie napisane na skanach, wówczas literówka **MUSI** zostać poprawiona.
+  
+#### Spacje
+
+Spacje pomiędzy dwoma znakami w danym ciągu powinny być zgodne z następującą konwencją (patrz na [#316][10]):
+
```
Comment (wopian) [inline CONTRIBUTING.pl.md:106]:
```suggestion

```

File: CONTRIBUTING.pl.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+Podobnie, jeśli słowo zawiera literówkę w digitalizacji, ale jest poprawnie napisane na skanach, wówczas literówka **MUSI** zostać poprawiona.
+  
+#### Spacje
+
+Spacje pomiędzy dwoma znakami w danym ciągu powinny być zgodne z następującą konwencją (patrz na [#316][10]):
+  
+- Pojedyńcza spacja dla nowego słowa.
+- Podwójna spacja dla nowego zdania.
+- Potrójna spacja dla akapitu.
+
```
Comment (wopian) [inline CONTRIBUTING.pl.md:110]:
```suggestion

```

File: CONTRIBUTING.pl.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+Typowe problemy, na które należy zwrócić uwagę podczas sprawdzania, obejmują między innymi:
+
+#### Literówki
+
+W niektórych miejscach pierwotni programiści popełnili błędy typograficzne podczas pisania komentarzy. Niektóre z nich zostały błędnie poprawione podczas wstępnej digitalizacji, jednak digitalizacja wprowadziła również błędy typograficzne, których nie było w skanach.
+
+Na przykład, jeśli zdigitalizowane komentarze zawierały słowo „SPACECRAFT”, ale na skanach odczytano „SPAECRAFT”, wówczas digitalizacja **MUSI** zostać poprawiona na „SPAECRAFT” (brakujące „C”).
+
+Podobnie, jeśli słowo zawiera literówkę w digitalizacji, ale jest poprawnie napisane na skanach, wówczas literówka **MUSI** zostać poprawiona.
+#### Spacje
```
Comment (wopian) [inline CONTRIBUTING.pl.md:102]:
```suggestion
Podobnie, jeśli słowo zawiera literówkę w digitalizacji, ale jest poprawnie napisane na skanach, wówczas literówka **MUSI** zostać poprawiona.

#### Spacje
```

File: CONTRIBUTING.pl.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+
+Spacje pomiędzy dwoma znakami w danym ciągu powinny być zgodne z następującą konwencją (patrz na [#316][10]):
+- Pojedyńcza spacja dla nowego słowa.
```
Comment (wopian) [inline CONTRIBUTING.pl.md:105]:
```suggestion
Spacje pomiędzy dwoma znakami w danym ciągu powinny być zgodne z następującą konwencją (patrz na [#316][10]):

- Pojedyńcza spacja dla nowego słowa.
```

File: CONTRIBUTING.pl.md [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+- Podwójna spacja dla nowego zdania.
+- Potrójna spacja dla akapitu.
+Nie wszystkie strony w skanach są zgodne z tym uogólnieniem, jeśli skany mają tylko jedną spację zamiast podwójnej, użyj jednej spacji.
```
Comment (wopian) [inline CONTRIBUTING.pl.md:108]:
```suggestion
- Potrójna spacja dla akapitu.

Nie wszystkie strony w skanach są zgodne z tym uogólnieniem, jeśli skany mają tylko jedną spację zamiast podwójnej, użyj jednej spacji.
```

**Comment by wopian** [comment]
Thank you!

**Pull request merged**